sound/soc/sof/sof-client.h

```diff
@@ -39,6 +39,10 @@ struct sof_client_dev {
 
 int sof_client_ipc_tx_message(struct sof_client_dev *cdev, void *ipc_msg,
 			      void *reply_data, size_t reply_bytes);
+static inline int sof_client_ipc_tx_message_no_reply(struct sof_client_dev *cdev, void *ipc_msg)
```

Comment:
@cujomalainey the change itself is OK but i think the name with the no_reply suffix is a bit misleading which seems to indicate that there's no reply for this IPC TX. It's not that there's no reply but that there's no data expected with the reply right?

Reply:
No the concept is that we don't care about the reply, only the return code. I.e don't fetch the reply data. Name was recommended by @plbossart

Reply:
@cujomalainey the return code that you mention here is part of the reply isnt it?

Reply:
Technically yes, but you do not need to fetch the data to get the return code. See error code fetch [here](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/ipc3.c#L299) but the copy out check is down [here](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/ipc3.c#L307) and this is what we are skipping

Reply:
Here is the [ipc4](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/ipc4.c#L304) equivalent code